packages/web-config/webpack.config.base.ts

```diff
@@ -84,6 +84,7 @@ export const getCommonConfig = () => {
                 [require.resolve('@babel/plugin-proposal-class-properties'), { loose: true }],
               ],
               presets: [
+                // the chrome version should be synced with npm/webpack-batteries-included-preprocessor/index.js
```

Comment:
...and npm/webpack-batteries-included-preprocessor/index.js should probably be synced with https://github.com/cypress-io/cypress/blob/6e702812f272f17e5e595a37d056447a54e6d37e/packages/server/lib/browsers/chrome.ts#L507-L510

is the minimum possible chrome version documented in a central location anywhere? i guess we can't publicly claim that we support chrome 63/64 though, since we only test on the latest chrome version AFAIK

Reply:
Interesting, I didn't know about this. Makes sense to bump the other ones to 64 then. 

I don't think this is documented at all and we don't have an official minimum version of Chrome. All we can say is we definitely don't support Chrome < 64, but yeah, can't guaranteed Chrome 64+ will work.